src/bilder/images/edxapp/files/edx_worker_playbook.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Deploy worker
+  hosts: all
+  become: true
+  gather_facts: true
+  vars:
+    COMMON_CONFIG_NO_LOGGING: false
+    # yamllint disable-line rule:line-length
+    EDXAPP_EDXAPP_SECRET_KEY: this-is-just-a-temporary-placeholder # pragma: allowlist secret
+    CREATE_SERVICE_WORKER_USERS: false
+  roles:
+  - role: edxapp
```

Comment:
?

Reply:
What's the question here? If it's wondering why the role is edxapp that's because that's how the workers get built. They deploy the edxapp repository and then configure the supervisor tasks to manage celery instead of setting up Nginx.